src/tests/test-data/parser/right-assoc.golden

```diff
@@ -0,0 +1 @@
+[0 [3 [1 2]]]
```

Comment:
@hjorthjort, as part of the review, please check the contents of these `.golden` files.